src/components/EventMap.js

```diff
@@ -14,6 +16,8 @@ export default function(store){
     data: {
       mapRef: null,
       eventsLayer: null,
+      initialCoordinates: [-96.9 , 37.8],
+      initialZoom: 4
```

Comment:
also: can you give me a few more details about how/when it is off? 

* Is the mis-centering happening only when the sidebar is open?
* Is the zoom level too zoomed-in or too zoomed-out?